site/src/pages/components/switch.explainer.mdx

```diff
@@ -300,12 +300,11 @@ Open UI research page: https://open-ui.org/components/switch/
 
 ### User interaction
 
-<div class="question">Should a swipe or drag&drop movement also toggle the switch?</div>
-
 From a user standpoint, the following interactions should be supported
 
 - Click/tap: toggles the switch
 - <kbd>Spacebar</kbd>: toggles the switch which has keyboard focus
+- Swipe: Switch should support gesture support akin to the range control ([#1045](https://github.com/openui/open-ui/issues/1045))
```

Comment:
@gregwhitworth wrote:
> Can you please link to the MDN documentation of the range control?

Here you go:

```suggestion
- Swipe: Switch should support gesture support akin to [input type="range"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range) ([#1045](https://github.com/openui/open-ui/issues/1045))
```

Reply:
@tantek thank you for the suggestion! Would you mind re-reviewing the PR?